packages/untp-test-suite/src/core/types/index.ts

```diff
@@ -14,6 +14,14 @@ export interface TestSuite {
   (credentialConfigsPath: string): Promise<TestSuiteResult[]>;
 }
 
-export interface TestSuiteResult extends ConfigContent {
-  errors: ErrorObject[] | null;
+export interface ICredentialConfig {
+  message: string;
+  keyword: string;
+  configPath: string;
 }
+
+export interface TestErrors {
+  errors: ErrorObject[] | ICredentialConfig | null;
```

Comment:
errors should accept an array of ICredentialConfig, because a credential config can be missing some fields then the errors should be an array for the credential config